build.gradle.kts

```diff
@@ -2,6 +2,7 @@ plugins {
     kotlin("jvm") version BuildPluginsVersion.KOTLIN apply false
     id("com.android.application") version BuildPluginsVersion.AGP apply false
     id("org.jlleitschuh.gradle.ktlint") version BuildPluginsVersion.KTLINT
+    id("org.jetbrains.kotlin.android") version "1.8.20" apply false
```

Comment:
hrm, do we need this one? I couldn't find where we are actually applying it?

If we still need it, let's use `BuildPluginsVersion.KOTLIN` for the version